server/forge/github/parse.go

```diff
@@ -148,28 +147,39 @@ func parseDeployHook(hook *github.DeploymentEvent) (*model.Repo, *model.Pipeline
 // parsePullHook parses a pull request hook and returns the Repo and Pipeline
 // details.
 func parsePullHook(hook *github.PullRequestEvent, merge bool) (*github.PullRequest, *model.Repo, *model.Pipeline, error) {
-	if hook.GetAction() != actionOpen && hook.GetAction() != actionSync && hook.GetAction() != actionClose {
-		return nil, nil, nil, nil
-	}
-
 	event := model.EventPull
-	if hook.GetPullRequest().GetState() == stateClose {
+	eventAction := ""
+
+	switch hook.GetAction() {
+	case actionOpen, actionReopen, actionSync:
```

Comment:
if reopen pull is merged https://github.com/woodpecker-ci/woodpecker/pull/5370

we let it as is ... else we have to add an filter that excludes this action from being parsed ...